hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -30,9 +30,9 @@
 	private boolean successFlag;
 	public boolean isSuccess() {return successFlag;}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)